lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -298,8 +298,7 @@ class GetExtentHelper(object):
     }
 
     def __init__(self, ax, direction):
-        if direction not in self._get_func_map:
-            raise KeyError("direction must be one of left, right, bottom, top")
```

Comment:
Technically, this is an API change because we now raise a `ValueError` instead of a `KeyError`.

Reply:
done